react-components/src/components/RevealToolbar/SettingsContainer/HighFidelityContainer.tsx

```diff
@@ -0,0 +1,68 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { Menu } from '@cognite/cogs.js';
+import { type ReactElement, useEffect, useMemo } from 'react';
+import { useReveal } from '../../RevealContainer/RevealContext';
+import { type ResolutionOptions } from '@cognite/reveal';
+import { type HighFidelityProps } from './types';
+
+export const HighFidelityContainer = ({
+  isHighFidelityMode,
+  setHighFidelityMode
+}: HighFidelityProps): ReactElement => {
+  const viewer = useReveal();
+
+  const defaultsQualityConfig = useMemo(() => {
+    return {
+      cadBudget: { ...viewer.cadBudget },
+      pointCloudBudget: { ...viewer.pointCloudBudget },
+      resolutionOptions: {
+        maxRenderResolution: 1.4e6,
+        movingCameraResolutionFactor: 1
```

Comment:
Can these things not be read out from the viewer?

Reply:
I could not find a way to access through viewer, it's not exposed from viewer